waspc/src/Generator.hs

```diff
@@ -1,27 +1,27 @@
 module Generator
        ( writeWebAppCode
-       , setup
+       , Generator.Setup.setup
+       , Generator.Start.start
        ) where
 
 import qualified Data.Text
 import qualified Data.Text.IO
-import Data.Time.Clock
-import qualified Paths_waspc
+import           Data.Time.Clock
 import qualified Data.Version
-import Control.Monad (mapM_)
-import qualified Path as P
-import System.Exit (ExitCode(..))
+import qualified Path                      as P
+import qualified Paths_waspc
 
-import StrongPath (Path, Abs, Dir, (</>))
-import qualified StrongPath as SP
-import CompileOptions (CompileOptions)
-import Wasp (Wasp)
-import Generator.WebAppGenerator (generateWebApp)
-import Generator.ServerGenerator (genServer)
-import Generator.DbGenerator (genDb)
-import qualified Generator.ServerGenerator.Setup
-import Generator.FileDraft (FileDraft, write)
-import Generator.Common (ProjectRootDir)
+import           CompileOptions            (CompileOptions)
+import           Generator.Common          (ProjectRootDir)
+import           Generator.DbGenerator     (genDb)
+import           Generator.FileDraft       (FileDraft, write)
+import           Generator.ServerGenerator (genServer)
+import qualified Generator.Setup
+import qualified Generator.Start
+import           Generator.WebAppGenerator (generateWebApp)
+import           StrongPath                (Abs, Dir, Path, (</>))
+import qualified StrongPath                as SP
+import           Wasp                      (Wasp)
```

Comment:
uu noice, much pretty. I have to set this up too.

Reply:
Yes you just need stylish-haskell!